ac/ac-autocode/src/ActivityRunner/Header.jsx

```diff
@@ -6,5 +6,6 @@ export default ({ config, style }: Object) => (
   <div style={style}>
     <h2>{config.title || 'Coding activity'}</h2>
     <h3>{config.guidelines || 'No guidelines were given for this exercice'}</h3>
+    <h3>{config.specifications || ''}</h3>
```

Comment:
Maybe `<p>` instead of `<h3>`?

If I understand correctly guidelines would be: "Write an algorithm to sort a list." and specifications would be: "There will be an input `a=[3,1,2]` (for example) and you should write the sorted list in a variable named `b`".  So guidelines are about the goal of the exercises while specifications are about the details of the implementation. Is that correct?